src/@types/styled-components.d.ts

```diff
@@ -22,6 +22,8 @@ declare module 'styled-components' {
         secondary: string;
         header: string;
         headerLight: string;
+        topline: string;
+        timestamp: string;
```

Comment:
I don't like the wording `Timestamp`. It's the human readable date and actually not limited to date. Timestamp suggests that I have to pass in a ISO date string or such while it's a simple string to display besides the title with dimmed colors. What about `dateFormatted`, `dateInfo` or such? (I have no explicit recommendation here though). Try out what sounds best for you from the perspective being a user of this component)